Scribe/ChangeHistory/Change.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using ParquetClassLibrary;
+
+namespace Scribe.ChangeHistory
+{
+    /// <summary>
+    /// Encapsulates a change request, providing both the ability to act on the request and to take it back.
+    /// </summary>
+    internal class Change
+    {
+        /// <summary>A summary of the action.</summary>
+        internal string Description;
+
+        /// <summary>State before change.</summary>
+        protected object OldState;
+
+        /// <summary>State after change.</summary>
+        protected object NewState;
+
+        /// <summary>The means to set the state in the backing store.</summary>
+        protected readonly Action<object> SetDatabaseValue;
+
+        /// <summary>The means to set the state in the backing store.</summary>
+        protected readonly Action<object> SetDisplayValue;
+
+        /// <summary>The means to set the state in the UI request-tracker.</summary>
+        protected readonly Action<object> SetOldValue;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Change"/> class.
+        /// </summary>
+        /// <param name="inOldState">State before change.</param>
+        /// <param name="inNewState">State after change.</param>
+        /// <param name="inControlName">Used in constructing a summary of the change.</param>
+        /// <param name="inSetDatabaseValue">The means to set the state in the backing store.</param>
+        /// <param name="inSetDisplayValue">The means to set the state in the backing store.</param>
```

Comment:
If the above comment is also duplicated, then this is. If it's not, then this is fine